Validation/EventGenerator/plugins/GenWeightValidation.cc

```diff
@@ -0,0 +1,177 @@
+#include "Validation/EventGenerator/interface/GenWeightValidation.h"
+
+#include "DataFormats/Math/interface/LorentzVector.h"
+#include "Validation/EventGenerator/interface/DQMHelper.h"
```

Comment:
This was already included in the header: please include it only where it is needed